week-4/Homework/mandatory/2-writers.js

```diff
@@ -42,3 +42,7 @@ let writers = [
 /*
 If you want an extra challenge, only `console.log()` the writers that are alive.
 */
+
+console.log(writers.map(i => `Hi, my name is ${i.firstName} ${i.lastName}. I am ${i.age} years old, and work as a ${i.occupation}.`))
+
+console.log(writers.filter(alive => alive.alive).map(alive => `${alive.firstName} is alive and ${alive.age} years old`))
```

Comment:
Great use of array methods! How about you try to name your variables a bit more semantically. Eg ```console.log(writers.filter(alive => alive.alive).map(alive => `${alive.firstName} is alive and ${alive.age} years old`))``` becomes
```console.log(writers.filter(writer => writer.alive).map(writer => `${writer.firstName} is alive and ${writer.age} years old`))```  😉